### ISSUE TITLE
Install based app

### ISSUE THREAD

**Issue by GYKgamer** [Botspot/pi-apps#2102]
### What is the name of the app?

Windows 95

### (Optional) Where is the app hosted?

https://github.com/felixrieseberg/windows95

### About the app

"Emulate" Windows 95 using JavaScript.

### Upload file or Add PR Link

https://github.com/felixrieseberg/windows95

### Confirmations

- [X] I have confirmed that this app is legal and not piracy.
- [X] I have confirmed that this app has never been discussed in https://github.com/Botspot/pi-apps/issues and https://github.com/Botspot/pi-apps/pulls, and it is not in the Pi Apps app list.
- [X] I have confirmed that this app can run on Raspberry Pi.

**Comment by Itai-Nelken** [comment]
While this is a cool app, it runs very poorly on the pi. Running windows 95/98 in qemu is much faster although a bit hatder to set up.
I made [bash scripts to create windows 98 and macos 9.2 vms on the pi using qemu](https://github.com/Itai-Nelken/RPi-QEMU-VM-scripts), I haven't tested them in a very long time, but they should work.

**Comment by theofficialgman** [comment]
I agree with Itai-Nelken, while a cool feat from a technical standpoint, running Win95 fully inside javascript and electron is a BAD way to get a useable experience on a low powered embedded system like the pi.

if someone makes a fully contained QEMU powered or other emulator driven win95 app, that would be better

**Comment by Botspot** [comment]
Also wanted to mention that back when I tried PINN (An OS-switcher alternative to NOOBS), there was an image that would boott to a dedicated Win98 QEMU VM. Compared to my previous attempts to get a VM working, I noticed this particulay VM was surprisingly snappy and responsive, given the same hardware. It might have even had networking set up - not sure.

It has been a long time and right now I don't have time to investigate further. I have a lot of personal matters to attend to at this time, so the time that I have will be mostly be spent on maintenance.

**Comment by virat2010** [comment]
I would love to see it! :D
